### PR TITLE
Redesign social page

### PR DESCRIPTION
**Pull request by Xelowak**
I changed the design of the social page and chirps, for it to be more visually appealing. 
Don't hesitate to make comments/suggestions. Note the screenshots have been taken on a 1440p screen, that's why it may appear quite small.
The "time ago" text needs to be in the same font as the rest, but since it is an external library, it's harder to do than I thought, so I'll probably do it later.

Here are screenshots of the new design:

Capture d’écran 2021-11-21 à 18 38 40

Capture d’écran 2021-11-21 à 18 31 14

In the future, these "person" icons will be replaced by images generated by each token.

### CODE REVIEW

**Review by jiabaow — DISMISSED** [review]
Thanks for the improvement!
LGTM but don't forget to update the snapshot as tests failed

**Comment by pierluca** [comment]
Do you want to resolve the merge conflicts? :)

**Comment by Xelowak** [comment]
Yes I will !

File: fe1-web/components/ChirpCard.tsx [hunk 1]
```diff
@@ -63,7 +105,7 @@ const ChirpCard = (props: IPropTypes) => {
 const propTypes = {
   sender: PropTypes.string.isRequired,
   text: PropTypes.string.isRequired,
-  time: PropTypes.instanceOf(Timestamp).isRequired,
+  time: PropTypes.number.isRequired,
```
Comment (pierluca) [inline fe1-web/components/ChirpCard.tsx:108]: Why not directly taking a chirp object as property?

Comment (Xelowak) [reply fe1-web/components/ChirpCard.tsx:108]: That's a really good remark, thanks a lot for the suggestion. I didn't think about it but it is much more practical.

**Review by pierluca — APPROVED** [review]
LGTM !

**Pull request merged**